### ISSUE TITLE
Contrib sub-package is missing in the mistletoe 1.0.0 PyPI version

### ISSUE THREAD

**Issue by franferrax** [miyuchina/mistletoe#177]
If I'm not misunderstanding #167, the `contrib` sub-package should be shipped as part of `mistletoe`, however it is missing from the [PyPI's `mistletoe==1.0.0` package](https://pypi.org/project/mistletoe/1.0.0), in either [mistletoe-1.0.0.tar.gz](https://files.pythonhosted.org/packages/8d/45/22f6dfa9f841b637b9ac893030a292fe47d5a88de3af1d5c59884a5de2c8/mistletoe-1.0.0.tar.gz) or [mistletoe-1.0.0-py3-none-any.whl](https://files.pythonhosted.org/packages/ea/81/14b1a5ec29a6016992a10e6569c61f10a3a06324102f9a05bab0770edc19/mistletoe-1.0.0-py3-none-any.whl).

## Steps to reproduce

```shell
virtualenv -q venv && ./venv/bin/pip3 install mistletoe==1.0.0
./venv/bin/python -c "from mistletoe.contrib.jira_renderer import JIRARenderer"
rm -rf venv
```

## Expected behavior

No errors occur

## Actual behavior

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'mistletoe.contrib'
```

**Comment by pbodnar** [comment]
@franferrax, thanks for reporting this issue. It looks like we need to list all exported bundles _explicitly_ here: https://github.com/miyuchina/mistletoe/blob/v1.0.0/setup.py#L11, i.e.:

```diff
- packages=['mistletoe'],
+ packages=['mistletoe', 'mistletoe.contrib'],
```

**Comment by pbodnar** [comment]
Fixed in 1.0.1 - and the version has just been released. :)

**Comment by franferrax** [comment]
Thanks a lot @pbodnar!
